pkg/internal/config/config_test.go

```diff
@@ -233,5 +236,6 @@ func getValidUserParameters() map[string]string {
 		config.TrafficGeneratorImageParamName:                      trafficGeneratorImage,
 		config.VMContainerDiskImageParamName:                       vmContainerDiskImage,
 		config.TestDurationParamName:                               testDuration,
+		config.VerboseParamName:                                    verboseTrue,
```

Comment:
Please consider using `strconv.FormatBool(true)`.
This will make `verboseTrue` redundant.

Reply:
DONE